securedrop-admin

```diff
@@ -5,10 +5,6 @@
 # the top level of the SecureDrop repository
 d=$(dirname "$0")
 if test "$1" = "setup" || test "$2" = "setup"; then
-   # the .venv symlink is to not confuse devs used to
-   # finding the directory at the top-level, prior to 0.6
-   rm -fr "$d/.venv"
-   ln -s "$d/admin/.venv" "$d/.venv"
```

Comment:
hmm... why was this removed?

Reply:
Because we want to keep the admin on Python 2 for now. I guess we could use .venv3 instead of .venv in the top level, for everything else, and rename everything once Python 2 is purged. I was just going down the path of using Python 3 by default and special-casing whatever we didn't want to migrate yet.

Reply:
while this change shouldn't be necessary to keep admins on Python 2 (since admins don't set up a virtualenv using the dev env logic, instead the logic in `securedrop-admin` only is used, and we don't have logic yet in that script to generate a python 3 virtualenv), it's fine to remove at this point. It's been a long time since 0.6 - worst case scenario that I'm aware of with this if there's an admin who hasn't run `securedrop-admin` since pre-0.6, is that they'll need to wait for `securedrop-admin setup` to re-set up the virtualenv.